docs/guide/actions.md

```diff
@@ -132,3 +132,23 @@ Textual supports the following builtin actions which are defined on the app.
 - [action_switch_screen][textual.app.App.action_switch_screen]
 - [action_screenshot][textual.app.App.action_screenshot]
 - [action_toggle_dark][textual.app.App.action_toggle_dark]
+
+## Temporarily suspending the application
```

Comment:
I think "App Basics" might be a better place for this. It's only tangentially related to actions.